scripts/whitelistedTokens.ts

```diff
@@ -0,0 +1,8 @@
+//stores the metadata related to staticATokens to be deployed on l2
```

Comment:
two remarks:
- we could move this file into constants/ folder
- it's now recommended to use `allowlist` instead of `whitelist`